src/makeRobotAccountant.js

```diff
@@ -6,7 +6,15 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return a => {
+    count++;
+
+    return b => count > 3 && count % 2 === 0
```

Comment:
It would be great to creat a variable for these conditions to make your code easier to read (e.g. isError)